ocaml/otherlibs/to_install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Shell scripts should have `set -eu` unless there is a good reason.

Reply:
Done.